parser/test.m

```diff
@@ -0,0 +1,40 @@
+function [output1, output2] = test(arg1, arg2)
+%TEST Computes output1 and output 2 from arg1 and arg2.
+%
+% SYNTAX
+%
+%   output1 = test(arg1, arg2)
+%   [output1, output2] = test(arg1, arg2)
+%
+% DESCRIPTION
+%
+%   Computes output1 as the sum of arg1 and arg2 and output2 as the
+%   difference between arg1 and arg2.
+%
+% INPUTS
+%
+%   arg1
+%     Scalar. This line is very long because i wanted to test if the input
+%     description would be correctly parsed if it is longer than one line.
+%
+%   arg2
+%     Scalar
+%
+% OUTPUTS
+%
+%   output1
+%     Sum of arg1 and arg2  
+%
+%   output2
+%     Difference between arg1 and arg2
+%
+% NOTES
+%
+% That function is destined to test the parsing of the function
+% documentation (done by parse_doc.m).
+
+output1 = arg1+arg2;
+% We don't want that kind of comments to appear in the parsed documentation.
+output2 = arg1-arg2; % Neither this kind of comments
+
```

Comment:
what are the assertions for this test? how can it fail?

Reply:
That test function is just some dummy  function that will be parsed (it could even be simply deleted and we could test the parsing directly on `parse_doc`). It is not a unit test. I would like to do an general unit test that includes the .md part once everything is good to go.

Reply:
bits-by-bits-- good practice is: for every code that is written, a test should go along it.

never "postpone" writing tests

things that we postpone might take a while to be implemented and then we forget about it

so the philosophy is: do it now, not tomorrow

also: a test that never fails is not a test

Reply:
Ok I'll create a unit test right now then. Is it ok if I create a `test` folder at the root of the repo ?

Reply:
@po09i is currently looking into this and can let us know what to do